### PR TITLE
Replace Key::Page(PageKey) with Key::Page{Up,Down}

### PR DESCRIPTION
**Pull request by joshka**
Saves 4 LOC

### CODE REVIEW

File: src/editor.rs [hunk 1]
```diff
@@ -729,7 +724,7 @@ impl Editor {
             self.file_name = None;
         }
         loop {
-            if let Some(mode) = self.prompt_mode.as_ref() {
+            if let Some(ref mode) = self.prompt_mode.as_ref() {
```
Comment (ilai-deutel) [inline src/editor.rs:727]: Is this needed? See clippy error in the PR checks

**Review by ilai-deutel — COMMENTED** [review]
Yay, thanks!

File: src/editor.rs (hunk 1 above)
Comment (joshka) [reply src/editor.rs:727]: oops - probably not - one or the other is needed. I think I was experimenting with it and didn't commit version with just `ref mode`

Comment (joshka) [reply src/editor.rs]: fe62086...f88e37f

**Comment by ilai-deutel** [comment]
@all-contributors-cli Please add @joshka for code

**Comment by ilai-deutel** [comment]
@all-contributors Please add @joshka for code

**Pull request merged**